subprojects/core/src/integTest/groovy/org/gradle/initialization/BuildInitializationBuildOperationsIntegrationTest.groovy

```diff
@@ -139,6 +139,7 @@ class BuildInitializationBuildOperationsIntegrationTest extends AbstractIntegrat
         }
 
         when:
+        executer.noDeprecationChecks()
```

Comment:
Do we need to keep track of these changed tests so we know which changes to revert once we fix the underlying deprecation?

Reply:
These are grouped as individual commits that we can just revert when we fix them in the follow up issues.